src/common/ApprovalsChip.tsx

```diff
@@ -25,11 +42,75 @@ export default function ApprovalsChip({ approvals }: { approvals: any }) {
     backgroundColor = '#4c8a4c'
   }
 
+  const getRequestResponses = (response) => {
+    let Icon
+    let secondaryText
+
+    if (response === 'Accepted') {
+      Icon = Done
+      secondaryText = 'Approved'
+    } else if (response === 'Declined') {
+      Icon = Close
+      secondaryText = 'Declined'
+    } else if (response === 'No Response') {
+      Icon = AccessTime
+      secondaryText = 'Awaiting response'
+    }
+
+    return [Icon, secondaryText]
+  }
+
+  const [ReviewerIcon, reviewerSecondaryText] = getRequestResponses(approvals?.reviewerResponse)
+  const reviewerPrimaryText = `Technical reviewer (${approvals?.reviewer})`
+
+  const [ManagerIcon, managerSecondaryText] = getRequestResponses(approvals?.managerResponse)
+  const managerPrimaryText = `Model manager (${approvals?.manager})`
+
+  const approvalsClicked = (event: MouseEvent) => {
```

Comment:
Convention for handling events is to name the handler as `handle<event description>` so in this case `handleApprovalsClicked`